sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2645,7 +2638,7 @@
         },
         "progress": {
           "format": "int32",
-          "description": "System generated.  Ignored at creation.\r\nRepresents the percentage of completion.",
+          "description": "System generated.  Ignored at creation. Represents the percentage of completion.",
```

Comment:
Instead of saying "System generated.  Ignored at creation." statement, we will make this field readonly.